src/packages/get-platform/pnpm-lock.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
```

Comment:
I think you can delete this lockfile? or does every new package comes with its own now?

Reply:
this needs to be checked in all pkg

Reply:
Yep will remove